src/commands/hosting-channel-deploy.ts

```diff
@@ -137,9 +160,18 @@ export default new Command("hosting:channel:deploy [channelId]")
         if (d.expireTime) {
           expires = `[expires ${bold(datetimeString(new Date(d.expireTime)))}]`;
         }
+        const versionPrefix = `sites/${d.target || d.site}/versions/`;
```

Comment:
I tested this out with a test project that has two sites and deploy targets set up for them. This line ends up breaking because `d.target` isn't what `d.site` is (`target` is the local alias for it). I think `.site` is the only thing that's needed here.

```suggestion
        const versionPrefix = `sites/${d.site}/versions/`;
```

Could you double check this fix? I think it should be fine, but it'd be nice for you to test as well :)

Reply:
Good catch. `version` is empty:

```
$ firebase --debug hosting:channel:deploy t1-test --project slam-home-api --json --only t1   
{
  "status": "success",
  "result": {
    "t1": {
      "site": "thing1",
      "target": "t1",
      "url": "https://thing1--t1-test-ltf4oe3y.web.app",
      "version": "",
      "expireTime": "2021-05-21T00:12:33.800426131Z"
    }
  }
}
```

Will fix.

Reply:
Fixed:

```
$ firebase --debug hosting:channel:deploy t1-test --project slam-home-api --json --only t1
{
  "status": "success",
  "result": {
    "t1": {
      "site": "thing1",
      "target": "t1",
      "url": "https://thing1--t1-test-ltf4oe3y.web.app",
      "version": "9af1bb9abd9d5d01",
      "expireTime": "2021-05-21T00:16:09.860986599Z"
    }
  }
}
```